[2023-09-04T06:01:46.196+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-02-06T00:00:00+00:00 [queued]>
[2023-09-04T06:01:46.211+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-02-06T00:00:00+00:00 [queued]>
[2023-09-04T06:01:46.212+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:01:46.319+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-02-06 00:00:00+00:00
[2023-09-04T06:01:46.329+0000] {standard_task_runner.py:57} INFO - Started process 4986 to run task
[2023-09-04T06:01:46.336+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-02-06T00:00:00+00:00', '--job-id', '292', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpqfiaj253']
[2023-09-04T06:01:46.339+0000] {standard_task_runner.py:85} INFO - Job 292: Subtask insert_into_table
[2023-09-04T06:01:46.416+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-02-06T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:01:46.588+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-02-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-06T00:00:00+00:00'
[2023-09-04T06:01:46.589+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-02-06', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:01:46.601+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:01:46.612+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:01:46.627+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-02-06', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:01:46.634+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:01:46.649+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220206T000000, start_date=20230904T060146, end_date=20230904T060146
[2023-09-04T06:01:46.707+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:01:46.728+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:04:35.787+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-02-06T00:00:00+00:00 [queued]>
[2023-09-04T06:04:35.816+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-02-06T00:00:00+00:00 [queued]>
[2023-09-04T06:04:35.818+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:04:35.981+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2022-02-06 00:00:00+00:00
[2023-09-04T06:04:35.998+0000] {standard_task_runner.py:57} INFO - Started process 6311 to run task
[2023-09-04T06:04:36.007+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'insert_into_table', 'scheduled__2022-02-06T00:00:00+00:00', '--job-id', '849', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp6l98yv2s']
[2023-09-04T06:04:36.012+0000] {standard_task_runner.py:85} INFO - Job 849: Subtask insert_into_table
[2023-09-04T06:04:36.170+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.insert_into_table scheduled__2022-02-06T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:04:36.440+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2022-02-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-06T00:00:00+00:00'
[2023-09-04T06:04:36.447+0000] {sql.py:274} INFO - Executing: 
            insert into dag_runs (dt, dag_id) values ('2022-02-06', 'dag_with_postgres_operator_v04')
        
[2023-09-04T06:04:36.469+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:36.495+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:36.525+0000] {sql.py:418} INFO - Running statement: 
            insert into dag_runs (dt, dag_id) values ('2022-02-06', 'dag_with_postgres_operator_v04')
        , parameters: None
[2023-09-04T06:04:36.535+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:04:36.576+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=insert_into_table, execution_date=20220206T000000, start_date=20230904T060435, end_date=20230904T060436
[2023-09-04T06:04:36.678+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:04:37.001+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
